docs/moment/00-use-it/10-system-js.md

```diff
@@ -0,0 +1,42 @@
+---
+title: System.js
+---
+
+To load moment, place it in the path specified by your System.config in the baseURL configuration. 
+Then import it into your page.
+
+```js
+<script src="system.js"></script>
+<script>
+  System.config({
+    baseURL: '/app'
+  });
+  
+  System.import('moment.js');
+ </script>
+ ```
+
+
+If you need moment to be loaded as global, you can do this with the meta configuration:
+
+```javascript
+System.config({
+  meta: {
+    'moment': { format: 'global' }
+  }
+});
+```
+
+Alternatively, if Moment isn't running as a global, you can configure moment to be a global for the global that is running:
```

Comment:
Yea, I think you're right. Will fix once I get docs building on windows again. Turns out I haven't rebuilt them in entirety since Tim changed everything over to a bash script :-)